app/controllers/fae/application_controller.rb

```diff
@@ -105,6 +108,12 @@ def first_user_redirect
       redirect_to fae.first_user_path if Fae::User.live_super_admins.blank?
     end
 
+    def setup_mfa_redirect
+      if @option.site_mfa_enabled && current_user.present? && current_user.user_mfa_enabled && !current_user.otp_required_for_login && request.path != new_two_factor_settings_path && request.path != two_factor_settings_path
```

Comment:
I hate this line, but its accurate to when we should redirect.  It forces the user to set up their MFA before they can go to any other page.  Would be super open to another way of getting this done.